src/main/java/duke/entity/task/Event.java

```diff
@@ -0,0 +1,80 @@
+package duke.entity.task;
+
+import duke.parser.DateTimeParser;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Event extends Task {
+
+    private String heldAt;
+    private Date heldAtDate;
+    private Date heldAtTime;
+
+    public Event(String taskName, String heldAt) {
+        super(taskName);
+        this.heldAt = heldAt;
+
+        DateTimeParser dateTimeParser = new DateTimeParser();
+        try {
+            this.heldAtDate = dateTimeParser.parseDate(heldAt);
+        } catch (ParseException e) {
+            this.heldAtDate = null;
+        }
+
+        try {
+            this.heldAtTime = dateTimeParser.parseTime(heldAt);
+        } catch (ParseException e) {
+            this.heldAtTime = null;
+        }
+    }
+
+    public String heldAt() {
+        return heldAt;
+    }
+
+    public void setHeldAt(String heldAt) {
+        this.heldAt = heldAt;
+    }
+
+    public String getHeldAt() {
+        return heldAt;
+    }
+
+    public Date getHeldAtDate() {
+        return heldAtDate;
+    }
+
+    public void setHeldAtDate(Date heldAtDate) {
+        this.heldAtDate = heldAtDate;
+    }
+
+    public Date getHeldAtTime() {
+        return heldAtTime;
+    }
+
+    public void setHeldAtTime(Date heldAtTime) {
+        this.heldAtTime = heldAtTime;
+    }
+
+    @Override
+    public String printTask() {
+        SimpleDateFormat timeFormat = new SimpleDateFormat("hh:mm aaa");
+        SimpleDateFormat dateFormat = new SimpleDateFormat("EEE, d MMM yyyy");
+
+        return "[E][" +  (super.isDone() ? "\u2713" : "\u2718" ) + "] "+ super.getTaskName() + " (at: " +
```

Comment:
Inappropriate styling for breaking lines. The `+` sign should be located in front of the new line, instead of being at the end of the previous line.